### PR TITLE
Pen and PDA related fixes

### PR DESCRIPTION
**Pull request by alsoandanswer**
fixes some grammar with PDAs.

clicking a fountain pen will make you twist the nib, allowing you to choose between cursive and normal writing.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -82,9 +82,9 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	if(..(user, 1))
 		to_chat(user, "The time [worldtime2text()] is displayed in the corner of the screen.")
 		if(id)
-			to_chat(user, "There is \a [id] in the ID slot.")
+			to_chat(user, "There is \a [id] in the ID slot. Alt-click to remove \the [id].")
 		if(inserted_item)
-			to_chat(user, "There is \a [inserted_item] in the pen slot.")
+			to_chat(user, "There is \a [inserted_item] in the pen slot. Ctrl-click to remove \the [inserted_item].")
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/PDA/PDA.dm]: I'd rather these be kept IC personally.

Comment (alsoandanswer) [reply code/game/objects/items/devices/PDA/PDA.dm]: Realistically, they'll only be seen by the user, and probably won't affect any RP

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -82,9 +82,9 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	if(..(user, 1))
 		to_chat(user, "The time [worldtime2text()] is displayed in the corner of the screen.")
 		if(id)
-			to_chat(user, "There is \a [id] in the ID slot.")
+			to_chat(user, "There is \a [id] in the ID slot. Alt-click to remove \the [id].")
```
Comment (Arrow768) [inline code/game/objects/items/devices/PDA/PDA.dm]: Not really a fan of giving usage descriptions via examine.

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 1 above)
Comment (Arrow768) [inline code/game/objects/items/devices/PDA/PDA.dm]: Not really a fan of giving usage descriptions via examine.

Comment (alsoandanswer) [reply code/game/objects/items/devices/PDA/PDA.dm]: moved to desc_fluff

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 2 above)
Comment (alsoandanswer) [reply code/game/objects/items/devices/PDA/PDA.dm]: moved to desc_fluff

**Comment by alsoandanswer** [comment]
!review

**Comment by Doxxmedearly** [comment]
Noticed that both on cargo tech and atmos tech, my PDAs aren't spawning with cartridges. If that's an easy enough fix to tack on to this PR, could you?

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 3]
```diff
@@ -6,6 +6,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 /obj/item/device/pda
 	name = "\improper PDA"
 	desc = "A portable microcomputer by Thinktronic Systems, LTD. Functionality determined by a preprogrammed ROM cartridge."
+	description_fluff = "Alt-click to remove IDs. Ctrl-click to remove things in the pen slot."
```
Comment (Alberyk) [inline code/game/objects/items/devices/PDA/PDA.dm]: description_info instead of description_fluff

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**